buzzmobile/process/agencynet/roadClassify.py

```diff
@@ -10,6 +10,8 @@
 parser.add_argument('iterations', type=int, help='Number of iterations')
 args = parser.parse_args()
 
+g = {} # globals
```

Comment:
There's something to be said about this being bad because we should be declaring exactly what things we use in the dict right after we instantiate it by setting them to None, but  we are not using agencynet right now and I don't care enough